tests/suite/credixLpDepositoryEditSuite.ts

```diff
@@ -6,12 +6,21 @@ import {
 } from '@uxd-protocol/uxd-client';
 import { getConnection, TXN_OPTS } from '../connection';
 import { editCredixLpDepositoryTest } from '../cases/editCredixLpDepositoryTest';
+import { createCredixLpDepositoryDevnetUSDC } from '../utils';
+
+export const credixLpDepositoryEditSuite = async function ({
+  authority,
+  controller,
+}: {
+  authority: Signer;
+  controller: Controller;
+}) {
+  let depository: CredixLpDepository;
+
+  before(async () => {
+    depository = await createCredixLpDepositoryDevnetUSDC();
```

Comment:
qq: why not passing the depository in parameter anymore?

Reply:
it's following the style of the original refactor by Greg,
I think the good thing is we keep the `Depository` class dependency on `-Test` but not on `-Suite`, easier for changes and create less merge conflicts potentially.
not sure is it your intention doing so @GregoryNEUT